packages/suite-native-atoms/src/VStack.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { ReactNode } from 'react';
+
+import { NativeStyleObject, prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { NativeSpacing } from '@trezor/theme';
+
+import { Box } from './Box';
+
+type VStackProps = {
+    children: ReactNode;
+    style?: NativeStyleObject;
+    spacing?: NativeSpacing;
+};
+
+type SpacerStyleProps = {
+    isLastChild: boolean;
+    spacing: NativeSpacing;
+};
+
+const spacerStyle = prepareNativeStyle<SpacerStyleProps>((_, { isLastChild, spacing }) => ({
+    extend: {
+        condition: !isLastChild,
+        style: {
+            marginBottom: spacing,
+        },
+    },
+}));
+
+export const VStack = ({ children, style, spacing = 'small' }: VStackProps) => {
```

Comment:
Please move this component `Demo` screen directly, it's useful for Demo purposes but won't need anything like this in real implementation.

Reply:
I think this could be used quite frequently. There will be lots of lists and stuff like that when you will need to specify margin bottom for each element of the list apart from the last one. It is quite frequent in all apps I have been working on as well as UI frameworks that I know. For example in Chakra UI this is used very oftern https://chakra-ui.com/docs/components/layout/stack#vstack as well as MUI https://mui.com/material-ui/react-stack/#main-content

I will remove it if you don't agree but I really think this is super useful and keeps this common UI pattern DRY. 

@bouchja1 What do you think?